pkgs/applications/misc/chataigne/default.nix

```
@@ -0,0 +1,34 @@
+{ lib, fetchurl, appimageTools, pkgs }:
```

```suggestion
{ lib, fetchurl, appimageTools, }:
```
please list the packages you use individually.